packages/shorebird_cli/lib/src/commands/release/release_android_command.dart

```diff
@@ -139,162 +140,165 @@ Use `shorebird flutter versions list` to list available versions.
       flutterRevisionForRelease = revision;
     }
 
-    final originalFlutterRevision = shorebirdEnv.flutterRevision;
-    final switchFlutterRevision =
-        flutterRevisionForRelease != originalFlutterRevision;
-
-    if (switchFlutterRevision) {
-      await shorebirdFlutter.useRevision(revision: flutterRevisionForRelease);
-    }
-
-    final flutterVersionString = await shorebirdFlutter.getVersionAndRevision();
-
-    final buildProgress = logger.progress(
-      'Building release with Flutter $flutterVersionString',
+    final flutterInstallProgress = logger.progress(
+      'Installing Flutter $flutterRevisionForRelease',
     );
+    await shorebirdFlutter.installRevision(revision: flutterRevisionForRelease);
+    flutterInstallProgress.complete();
```

Comment:
Minor but if `installRevision` throws, then the progress will never complete which will look a bit weird to users.

Reply:
On it

Reply:
Fixed